plutus-ledger-api/src/PlutusLedgerApi/V1/Scripts.hs

```diff
@@ -223,50 +223,104 @@ newtype StakeValidator = StakeValidator { getStakeValidator :: Script }
 instance Haskell.Show StakeValidator where
     show = const "StakeValidator { <script> }"
 
--- | Script runtime representation of a @Digest SHA256@.
+{- | Script runtime representation of a 28 byte hash. Hashed with /BLAKE2b-224/.
```

Comment:
Perhaps it's closer to say it's the "Type representing a hash of a script"? And then similarly below but for hashes of validators, datums, etc. That's the main purpose of the type, to help you remember what kind of thing it's a hash of.